pure_pursuit_jerk_wrapper/launch/pure_pursuit_jerk_wrapper.launch

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<launch>
+    <remap from="final_waypoints" to="carma_final_waypoints"/>
```

Comment:
Looks like this wrapper is using the same pure_pursuit instance as the other wrapper. This launch file should probably launch a seperate instance of the pure_pursuit node with a different input topic so that there is less risk of a race condition causing overlapping commands. But if @kjrush wants to wait I imagine its a bit of a low risk at the moment.